plugins/dbviewer/frontend/public/javascripts/countly.views.js

```diff
@@ -313,19 +364,9 @@ window.DBViewerView = countlyView.extend({
 				// prepare filter by input values
 				var filter = $(".dbviewer-collection-filter").val() == "" ? JSON.stringify({}) : $(".dbviewer-collection-filter").val();
 				// prepare sort by input values
-				var sort = {};
-				if (store.get('dbviewer_sort_show') && $('#dbviewer-sort_param').val() !== "") {
-					sort[$("#dbviewer-sort_param").val()] = parseInt($('#dbviewer-sort_type').val());
-					store.set('dbviewer_sort_value', $('#dbviewer-sort_param').val());
-					store.set('dbviewer_sort_type', $('#dbviewer-sort_type').val());
-				} else {
-					self.sort = {};
-					store.remove('dbviewer_sort_value');
-				}
-
 				self.filter = filter;
 				self.projection = JSON.stringify(projection);
-				self.sort = JSON.stringify(sort);
+				self.sort = JSON.stringify(self.sort);
```

Comment:
This kind of assignment always worries me, what if in some other place self.sort is expected to be json string and in other json object?